### ISSUE TITLE
Add configurable URLS for the validator service and NERVE in CWRC-Writer. 

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-WriterBase#209]

**Comment by ajmacdonald** [comment]
There's some work done towards this already: https://github.com/cwrc/CWRC-GitWriter/commit/26e8ccb337076d7a85d614846fcf31ea605fe41d

**Comment by lucaju** [comment]
Yup. I saw these changes yesterday night when I pull the changes @ajmacdonald made it. I guess these changes solve this issue.

**Comment by ilovan** [comment]
Any ideas how to test this from the front end (if at all possible)?

**Comment by lucaju** [comment]
This is defined in the configure file. It does not has any indication of these URL in the Frontend. We could put them in settings panel, in the future.

We know that it is working because dev-server, my local dev, and @ajmacdonald local dev all point to different urls (self-hosted in the first two, and some place else in Andrews end)

**Comment by ilovan** [comment]
sounds good then
